z_infra_provisioning/cloud_infra_center/ocp_upi/roles/configure-installer-client/tasks/main.yaml

```diff
@@ -10,9 +10,19 @@
 ---
 # tasks file for configure-installer-and-image
 
+- name: Verify openshift minor version
+  shell: |
+    version=$(curl -s https://mirror.openshift.com/pub/openshift-v4/clients/ocp/{{ openshift_version }}.{{ openshift_minor_version }}/release.txt | grep 'Name:' | awk -F: '{print $2}' | xargs)
+    if [ -z $version ]; then
+      echo $(curl -s https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest-{{ openshift_version }}/release.txt | grep 'Name:' | awk -F: '{print $2}' | xargs)
```

Comment:
same as above. The link should involve {ansible_architecture}, such as:
https://mirror.openshift.com/pub/openshift-v4/s390x/clients/ocp/latest-4.11/release.txt